Docs/source/developers/performance_tests.rst

```diff
@@ -0,0 +1,101 @@
+.. _developers-performance_tests:
+
+Automated performance tests
+===========================
+
+WarpX has automated performance test scripts, which run weak scalings for various tests on a weekly basis. The results are stored in the `perf_logs repo <https://github.com/ECP-WarpX/perf_logs>`_ and plots of the performance history can be found on `this page <https://ecp-warpx.github.io/perf_logs/>`_.
+
+These performance tests run automatically, so they need to do ``git`` operations etc. For this reason, they need a separate clone of the source repos, so they don't conflict with one's usual operations. This is typically in a sub-directory in the ``$HOME``, with variable ``$AUTOMATED_PERF_TESTS`` pointing to it. Similarly, a directory is needed to run the simulations and store the results. By default, it is ``$SCRATCH/performance_warpx``.
+
+The test runs a weak scaling (1,2,8,64,256,512 nodes) for 6 different tests ``Tools/PerformanceTests/automated_test_{1,2,3,4,5,6}_*``, gathered in 1 batch job per number of nodes to avoid submitting too many jobs.
+
+Setup on Summit @ OLCF
+----------------------
+
+Here's my setup on Summit, feel free to adapt it to your needs:
+
+.. code-block:: sh
+
+   mkdir $HOME/AUTOMATED_PERF_TESTS
+   export AUTOMATED_PERF_TESTS=$HOME/AUTOMATED_PERF_TESTS/ # I put it in $HOME/my_bashrc.sh
+   export SCRATCH=/gpfs/alpine/scratch/<user name>/aph114/ # I put it in $HOME/my_bashrc.sh
+   cd $AUTOMATED_PERF_TESTS
+   git clone https://github.com/ECP-WarpX/WarpX.git warpx
+   git clone https://github.com/ECP-WarpX/picsar.git
+   git clone --branch development https://github.com/AMReX-Codes/amrex.git
+   git clone https://github.com/ECP-WarpX/perf_logs.git
+
+Then, in ``$AUTOMATED_PERF_TESTS``, create a file ``run_automated_performance_tests_512.sh`` with the following content:
+
+.. code-block:: sh
+
+   #!/bin/bash -l
+   #BSUB -P APH114
+   #BSUB -W 00:10
+   #BSUB -nnodes 1
+   #BSUB -J PERFTEST
+   #BSUB -e err_automated_tests.txt
+   #BSUB -o out_automated_tests.txt
+
+   source $HOME/my_bashrc.sh
+
+   # Make sure all dependencies are install (in $HOME/.local/)
+   cd $HOME
+   module load python/3.7.0-anaconda3-5.3.0
+   pip install numpy==1.15.4 --user
+   pip install pandas --user
+   pip install bokeh --user
+   pip install gitpython --user
+
+   # Check where numpy is installed, and prepend its parent directory to PYTHONPATH
+   home_to_numpy=$(python -c "import numpy ; import os ; print(os.path.split(os.path.split(numpy.__file__)[0])[0])")
```

Comment:
Interesting, is this still needed? If you can `import` numpy it means it is already in a PYTHONPATH or python system path.
```suggestion
   home_to_numpy=$(python3 -c "import numpy ; import os ; print(os.path.split(os.path.split(numpy.__file__)[0])[0])")
```

Reply:
This is still required for me, but I am not sure why.